plugins/module_utils/network/iosxr/argspec/ping/ping.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the
+# cli_rm_builder.
+#
+# Manually editing this file is not advised.
+#
+# To update the argspec make the desired changes
+# in the module docstring and re-run
+# cli_rm_builder.
+#
+#############################################
+
+"""
+The arg spec for the iosxr_ping module
+"""
+
+
+class PingArgs(object):  # pylint: disable=R0903
+    """The arg spec for the iosxr_ping module"""
+
+    argument_spec = {
+        "count": {"type": "int"},
```

Comment:
Is there a `timeout` option in XR?

Reply:
I don't see any timeout native to the ping command